site.yml

```diff
@@ -187,7 +187,6 @@
 
 - name: ceph osds
   hosts: ceph_osds
-  serial: 1
```

Comment:
there are many cases. If I make full case tests now, I will have to do it again before optimization PR.
I would like to do full case tests before the last feature PR.
For now, I just cover the base cases.